### PR TITLE
RK-10928 - Go rook bump to 0.0.5

### PR DESCRIPTION
**Pull request by ElDuderinos**
Aligning with proper go get modules and using an updated version.
Going back to alpine to image alignment.
Tested locally with docker compose.

### CODE REVIEW

**Review by dan505512 — DISMISSED** [review]

**Comment by lirook** [comment]
Tried to run "make build", got this:
```
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DataExtractor.cpp.o): in function `llvm::format_object<unsigned long, char const*>::snprint(char*, unsigned int) const':
#13 22.69 DataExtractor.cpp:(.text._ZNK4llvm13format_objectIJmPKcEE7snprintEPcj[_ZNK4llvm13format_objectIJmPKcEE7snprintEPcj]+0x27): undefined reference to `__snprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DataExtractor.cpp.o): in function `llvm::format_object<unsigned long, unsigned long, unsigned long>::snprint(char*, unsigned int) const':
#13 22.69 DataExtractor.cpp:(.text._ZNK4llvm13format_objectIJmmmEE7snprintEPcj[_ZNK4llvm13format_objectIJmmmEE7snprintEPcj]+0x27): undefined reference to `__snprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(ItaniumManglingCanonicalizer.cpp.o): in function `llvm::itanium_demangle::FloatLiteralImpl<long double>::printLeft(llvm::itanium_demangle::OutputStream&) const':
#13 22.69 ItaniumManglingCanonicalizer.cpp:(.text._ZNK4llvm16itanium_demangle16FloatLiteralImplIeE9printLeftERNS0_12OutputStreamE[_ZNK4llvm16itanium_demangle16FloatLiteralImplIeE9printLeftERNS0_12OutputStreamE]+0xb4): undefined reference to `__snprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(ItaniumManglingCanonicalizer.cpp.o): in function `llvm::itanium_demangle::FloatLiteralImpl<double>::printLeft(llvm::itanium_demangle::OutputStream&) const':
#13 22.69 ItaniumManglingCanonicalizer.cpp:(.text._ZNK4llvm16itanium_demangle16FloatLiteralImplIdE9printLeftERNS0_12OutputStreamE[_ZNK4llvm16itanium_demangle16FloatLiteralImplIdE9printLeftERNS0_12OutputStreamE]+0xa8): undefined reference to `__snprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(ItaniumManglingCanonicalizer.cpp.o): in function `llvm::itanium_demangle::FloatLiteralImpl<float>::printLeft(llvm::itanium_demangle::OutputStream&) const':
#13 22.69 ItaniumManglingCanonicalizer.cpp:(.text._ZNK4llvm16itanium_demangle16FloatLiteralImplIfE9printLeftERNS0_12OutputStreamE[_ZNK4llvm16itanium_demangle16FloatLiteralImplIfE9printLeftERNS0_12OutputStreamE]+0xb0): undefined reference to `__snprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(PrettyStackTrace.cpp.o): in function `llvm::PrettyStackTraceFormat::PrettyStackTraceFormat(char const*, ...)':
#13 22.69 PrettyStackTrace.cpp:(.text._ZN4llvm22PrettyStackTraceFormatC2EPKcz+0xe3): undefined reference to `__vsnprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: PrettyStackTrace.cpp:(.text._ZN4llvm22PrettyStackTraceFormatC2EPKcz+0x172): undefined reference to `__vsnprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(SHA1.cpp.o): in function `llvm::SHA1::hash(llvm::ArrayRef<unsigned char>)':
#13 22.69 SHA1.cpp:(.text._ZN4llvm4SHA14hashENS_8ArrayRefIhEE+0x5f): undefined reference to `__memcpy_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(regerror.c.o): in function `llvm_regerror':
#13 22.69 regerror.c:(.text.llvm_regerror+0x104): undefined reference to `__snprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(InlineAsm.cpp.o): in function `llvm::hash_code llvm::hash_combine<llvm::StringRef, llvm::StringRef, bool, bool, llvm::InlineAsm::AsmDialect, llvm::FunctionType*>(llvm::StringRef const&, llvm::StringRef const&, bool const&, bool const&, llvm::InlineAsm::AsmDialect const&, llvm::FunctionType* const&)':
#13 22.69 InlineAsm.cpp:(.text._ZN4llvm12hash_combineIJNS_9StringRefES1_bbNS_9InlineAsm10AsmDialectEPNS_12FunctionTypeEEEENS_9hash_codeEDpRKT_[_ZN4llvm12hash_combineIJNS_9StringRefES1_bbNS_9InlineAsm10AsmDialectEPNS_12FunctionTypeEEEENS_9hash_codeEDpRKT_]+0x184): undefined reference to `__memcpy_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: InlineAsm.cpp:(.text._ZN4llvm12hash_combineIJNS_9StringRefES1_bbNS_9InlineAsm10AsmDialectEPNS_12FunctionTypeEEEENS_9hash_codeEDpRKT_[_ZN4llvm12hash_combineIJNS_9StringRefES1_bbNS_9InlineAsm10AsmDialectEPNS_12FunctionTypeEEEENS_9hash_codeEDpRKT_]+0x213): undefined reference to `__memcpy_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DWARFDebugAddr.cpp.o): in function `llvm::format_object<unsigned int, unsigned long>::snprint(char*, unsigned int) const':
#13 22.69 DWARFDebugAddr.cpp:(.text._ZNK4llvm13format_objectIJjmEE7snprintEPcj[_ZNK4llvm13format_objectIJjmEE7snprintEPcj]+0x27): undefined reference to `__snprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DWARFDebugAddr.cpp.o): in function `llvm::format_object<unsigned long, unsigned char, unsigned char>::snprint(char*, unsigned int) const':
#13 22.69 DWARFDebugAddr.cpp:(.text._ZNK4llvm13format_objectIJmhhEE7snprintEPcj[_ZNK4llvm13format_objectIJmhhEE7snprintEPcj]+0x2b): undefined reference to `__snprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DWARFDebugAddr.cpp.o): in function `llvm::format_object<unsigned long, unsigned short>::snprint(char*, unsigned int) const':
#13 22.69 DWARFDebugAddr.cpp:(.text._ZNK4llvm13format_objectIJmtEE7snprintEPcj[_ZNK4llvm13format_objectIJmtEE7snprintEPcj]+0x29): undefined reference to `__snprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DWARFDebugAddr.cpp.o): in function `llvm::format_object<unsigned long, unsigned long, unsigned char>::snprint(char*, unsigned int) const':
#13 22.69 DWARFDebugAddr.cpp:(.text._ZNK4llvm13format_objectIJmmhEE7snprintEPcj[_ZNK4llvm13format_objectIJmmhEE7snprintEPcj]+0x29): undefined reference to `__snprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DWARFDebugAddr.cpp.o): in function `llvm::format_object<unsigned long, unsigned char>::snprint(char*, unsigned int) const':
#13 22.69 DWARFDebugAddr.cpp:(.text._ZNK4llvm13format_objectIJmhEE7snprintEPcj[_ZNK4llvm13format_objectIJmhEE7snprintEPcj]+0x29): undefined reference to `__snprintf_chk'
#13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DWARFDebugArangeSet.cpp.o):DWARFDebugArangeSet.cpp:(.text._ZNK4llvm13format_objectIJjjmEE7snprintEPcj[_ZNK4llvm13format_objectIJjjmEE7snprintEPcj]+0x28): more undefined references to `__snprintf_chk' follow
#13 22.69 collect2: error: ld returned 1 exit status
------
executor failed running [/bin/sh -c go build -gcflags='-N -l' -o /go/bin/productcatalogservice .]: exit code: 2
make: *** [build] Error 1
```

**Comment by ElDuderinos** [comment]
> Tried to run "make build", got this:
> 
> ```
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DataExtractor.cpp.o): in function `llvm::format_object<unsigned long, char const*>::snprint(char*, unsigned int) const':
> #13 22.69 DataExtractor.cpp:(.text._ZNK4llvm13format_objectIJmPKcEE7snprintEPcj[_ZNK4llvm13format_objectIJmPKcEE7snprintEPcj]+0x27): undefined reference to `__snprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DataExtractor.cpp.o): in function `llvm::format_object<unsigned long, unsigned long, unsigned long>::snprint(char*, unsigned int) const':
> #13 22.69 DataExtractor.cpp:(.text._ZNK4llvm13format_objectIJmmmEE7snprintEPcj[_ZNK4llvm13format_objectIJmmmEE7snprintEPcj]+0x27): undefined reference to `__snprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(ItaniumManglingCanonicalizer.cpp.o): in function `llvm::itanium_demangle::FloatLiteralImpl<long double>::printLeft(llvm::itanium_demangle::OutputStream&) const':
> #13 22.69 ItaniumManglingCanonicalizer.cpp:(.text._ZNK4llvm16itanium_demangle16FloatLiteralImplIeE9printLeftERNS0_12OutputStreamE[_ZNK4llvm16itanium_demangle16FloatLiteralImplIeE9printLeftERNS0_12OutputStreamE]+0xb4): undefined reference to `__snprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(ItaniumManglingCanonicalizer.cpp.o): in function `llvm::itanium_demangle::FloatLiteralImpl<double>::printLeft(llvm::itanium_demangle::OutputStream&) const':
> #13 22.69 ItaniumManglingCanonicalizer.cpp:(.text._ZNK4llvm16itanium_demangle16FloatLiteralImplIdE9printLeftERNS0_12OutputStreamE[_ZNK4llvm16itanium_demangle16FloatLiteralImplIdE9printLeftERNS0_12OutputStreamE]+0xa8): undefined reference to `__snprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(ItaniumManglingCanonicalizer.cpp.o): in function `llvm::itanium_demangle::FloatLiteralImpl<float>::printLeft(llvm::itanium_demangle::OutputStream&) const':
> #13 22.69 ItaniumManglingCanonicalizer.cpp:(.text._ZNK4llvm16itanium_demangle16FloatLiteralImplIfE9printLeftERNS0_12OutputStreamE[_ZNK4llvm16itanium_demangle16FloatLiteralImplIfE9printLeftERNS0_12OutputStreamE]+0xb0): undefined reference to `__snprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(PrettyStackTrace.cpp.o): in function `llvm::PrettyStackTraceFormat::PrettyStackTraceFormat(char const*, ...)':
> #13 22.69 PrettyStackTrace.cpp:(.text._ZN4llvm22PrettyStackTraceFormatC2EPKcz+0xe3): undefined reference to `__vsnprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: PrettyStackTrace.cpp:(.text._ZN4llvm22PrettyStackTraceFormatC2EPKcz+0x172): undefined reference to `__vsnprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(SHA1.cpp.o): in function `llvm::SHA1::hash(llvm::ArrayRef<unsigned char>)':
> #13 22.69 SHA1.cpp:(.text._ZN4llvm4SHA14hashENS_8ArrayRefIhEE+0x5f): undefined reference to `__memcpy_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(regerror.c.o): in function `llvm_regerror':
> #13 22.69 regerror.c:(.text.llvm_regerror+0x104): undefined reference to `__snprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(InlineAsm.cpp.o): in function `llvm::hash_code llvm::hash_combine<llvm::StringRef, llvm::StringRef, bool, bool, llvm::InlineAsm::AsmDialect, llvm::FunctionType*>(llvm::StringRef const&, llvm::StringRef const&, bool const&, bool const&, llvm::InlineAsm::AsmDialect const&, llvm::FunctionType* const&)':
> #13 22.69 InlineAsm.cpp:(.text._ZN4llvm12hash_combineIJNS_9StringRefES1_bbNS_9InlineAsm10AsmDialectEPNS_12FunctionTypeEEEENS_9hash_codeEDpRKT_[_ZN4llvm12hash_combineIJNS_9StringRefES1_bbNS_9InlineAsm10AsmDialectEPNS_12FunctionTypeEEEENS_9hash_codeEDpRKT_]+0x184): undefined reference to `__memcpy_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: InlineAsm.cpp:(.text._ZN4llvm12hash_combineIJNS_9StringRefES1_bbNS_9InlineAsm10AsmDialectEPNS_12FunctionTypeEEEENS_9hash_codeEDpRKT_[_ZN4llvm12hash_combineIJNS_9StringRefES1_bbNS_9InlineAsm10AsmDialectEPNS_12FunctionTypeEEEENS_9hash_codeEDpRKT_]+0x213): undefined reference to `__memcpy_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DWARFDebugAddr.cpp.o): in function `llvm::format_object<unsigned int, unsigned long>::snprint(char*, unsigned int) const':
> #13 22.69 DWARFDebugAddr.cpp:(.text._ZNK4llvm13format_objectIJjmEE7snprintEPcj[_ZNK4llvm13format_objectIJjmEE7snprintEPcj]+0x27): undefined reference to `__snprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DWARFDebugAddr.cpp.o): in function `llvm::format_object<unsigned long, unsigned char, unsigned char>::snprint(char*, unsigned int) const':
> #13 22.69 DWARFDebugAddr.cpp:(.text._ZNK4llvm13format_objectIJmhhEE7snprintEPcj[_ZNK4llvm13format_objectIJmhhEE7snprintEPcj]+0x2b): undefined reference to `__snprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DWARFDebugAddr.cpp.o): in function `llvm::format_object<unsigned long, unsigned short>::snprint(char*, unsigned int) const':
> #13 22.69 DWARFDebugAddr.cpp:(.text._ZNK4llvm13format_objectIJmtEE7snprintEPcj[_ZNK4llvm13format_objectIJmtEE7snprintEPcj]+0x29): undefined reference to `__snprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DWARFDebugAddr.cpp.o): in function `llvm::format_object<unsigned long, unsigned long, unsigned char>::snprint(char*, unsigned int) const':
> #13 22.69 DWARFDebugAddr.cpp:(.text._ZNK4llvm13format_objectIJmmhEE7snprintEPcj[_ZNK4llvm13format_objectIJmmhEE7snprintEPcj]+0x29): undefined reference to `__snprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DWARFDebugAddr.cpp.o): in function `llvm::format_object<unsigned long, unsigned char>::snprint(char*, unsigned int) const':
> #13 22.69 DWARFDebugAddr.cpp:(.text._ZNK4llvm13format_objectIJmhEE7snprintEPcj[_ZNK4llvm13format_objectIJmhEE7snprintEPcj]+0x29): undefined reference to `__snprintf_chk'
> #13 22.69 /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: /go/pkg/mod/github.com/!rookout/!go!rook@v0.0.5/pkg/services/instrumentation/hooker/libhook_lib_pack_ubuntu20.a(DWARFDebugArangeSet.cpp.o):DWARFDebugArangeSet.cpp:(.text._ZNK4llvm13format_objectIJjjmEE7snprintEPcj[_ZNK4llvm13format_objectIJjjmEE7snprintEPcj]+0x28): more undefined references to `__snprintf_chk' follow
> #13 22.69 collect2: error: ld returned 1 exit status
> ------
> executor failed running [/bin/sh -c go build -gcflags='-N -l' -o /go/bin/productcatalogservice .]: exit code: 2
> make: *** [build] Error 1
> ```

I'm not sure what you're trying to do and what are you referring to.
But from your log I see you're running in alpine and we're trying to link to the ubuntu20 lib.
So either run in ubuntu (not recommended) or just use the alpine314 tag

File: src/productcatalogservice/Dockerfile [hunk 1]
```diff
@@ -12,32 +12,24 @@
...
 WORKDIR /src
-
-RUN git clone --depth 1 https://${SONARIO_GH_TOKEN}@github.com/Rookout/GoRook.git -b v0.0.5
-
-# restore dependencies
-COPY go.mod go.sum ./
-RUN go mod download
 COPY . .
+RUN go mod download
 RUN go build -gcflags='-N -l' -o /go/bin/productcatalogservice .
```
Comment (lirook) [inline src/productcatalogservice/Dockerfile:29]:
```suggestion
RUN go build -tags=${GOROOK_OS_TAG} -gcflags='-N -l' -o /go/bin/productcatalogservice .
```

**Review by lirook — CHANGES_REQUESTED** [review]

File: src/shippingservice/Dockerfile [hunk 2]
```diff
@@ -12,31 +12,25 @@
...
-
-RUN git clone --depth 1 https://${SONARIO_GH_TOKEN}@github.com/Rookout/GoRook.git -b v0.0.5
-
-# restore dependencies
-COPY go.mod go.sum ./
-RUN go mod download
 COPY . .
+RUN go mod download
+
 RUN go build -gcflags='-N -l' -o /go/bin/shippingservice
```
Comment (lirook) [inline src/shippingservice/Dockerfile:30]:
```suggestion
RUN go build -tags=${GOROOK_OS_TAG} -gcflags='-N -l' -o /go/bin/shippingservice
```

**Review by lirook — CHANGES_REQUESTED** [review]

File: src/productcatalogservice/Dockerfile (hunk 1 above)
Comment (ElDuderinos) [reply src/productcatalogservice/Dockerfile:29]: yep, I missed those

**Review by lirook — APPROVED** [review]

**Pull request merged**